### PR TITLE
Add ability to remove images

### PR DESCRIPTION
**Pull request by zach-taylor**
# What
This adds the ability to remove an image from an instance. I have added a small x button next to each image. Upon clicking, the x is turned into a spinning spinner.

screen shot 2015-03-28 at 1 02 34 am

# Outstanding issues
* I am running into an issue where the image list won't refresh after deleting an image. It seems the browser begins the GET request but is stuck in the (pending) state.
screen shot 2015-03-28 at 1 10 50 am

### CODE REVIEW

**Comment by theMagicalKarp** [comment]
+1

File: app/js/instances/instanceDetailController.js [hunk 1]
```diff
@@ -83,6 +81,14 @@ function instanceDetailController($scope, $routeParams, $timeout, flux, dockerSe
             }
         });
     };
+
+    $scope.removeImage = function(image) {
+        image.loading = true;
+        dockerService.d('images.remove', {
+            host: $routeParams.host,
```
Comment (zach-taylor) [inline app/js/instances/instanceDetailController.js]: @ngmiller I wasn't able to use $scope.instance.alias here for some reason. It was `undefined` for some reason.

Comment (ngmiller) [inline app/js/instances/instanceDetailController.js]: Hmmm that's odd. I was able to get valid output with this. Double check the spelling :)

```
$scope.removeImage = function(image) {
        console.log($scope.instance);
        console.log($scope.instance.alias);

        image.loading = true;
        ....
```

Comment (zach-taylor) [inline app/js/instances/instanceDetailController.js]: Hm, yeah, it's working for me now too. Must have been something else then.

**Comment by theMagicalKarp** [comment]
+1

**Comment by zach-taylor** [comment]
@ngmiller build is passing now. Travis was having issues cloning from GitHub due to the DDoS attack. Also, do you have any insight on that outstanding issue I'm seeing above? Something's off for my setup because I'm also not getting the status bar when adding an image via the UI. I'm running both Lighthouse and Beacon in Docker containers on boot2docker.

**Comment by ngmiller** [comment]
@zach-taylor Not sure what's going on with the pending GET request - I wasn't having that issue so I just assumed it was fixed :) Best thing to do for now would be to make sure you have the most recent build of beacon to get the streaming functionality (i.e. do a `docker pull lighthouse/beacon:latest`) I'm thinking it might be that but I can't say for sure. Feel free to hit me up on chat if you're still running into issues, otherwise I'd be happy to take a look during our meeting as well

**Comment by zach-taylor** [comment]
@ngmiller this is working for me now. should we do the error handling stuff in a different PR?

**Comment by ngmiller** [comment]
Sounds good to me +1

**Pull request merged**